Turf/Turf.swift

```diff
@@ -4,10 +4,9 @@ public typealias LocationRadians = Double
 public typealias RadianDistance = Double
 public typealias RadianDirection = Double
 
-struct Constants {
-    static let metersPerRadian = 6_373_000.0
-    static let equatorialRadius:Double = 6378137
-}
+
+let metersPerRadian: CLLocationDistance = 6_373_000.0
+let equatorialRadius: CLLocationDistance = 6378137
```

Comment:
`equatorialRadius` is currently set to the equatorial radius according to WGS84 and the [IUGG](https://github.com/Project-OSRM/osrm-backend/blob/088d4edc6b761d2642a9b1ab7d85b701d3165fb2/include/util/coordinate_calculation.hpp#L26-L28). This value is used in [OSRM](https://github.com/Project-OSRM/osrm-backend/blob/1d4d3b80b5e53fb342a73b5556c9dd3e78d5c4c7/include/util/web_mercator.hpp#L19) and [mbgl](https://github.com/mapbox/mapbox-gl-native/blob/0fddad6e0b828af2d49d65953d31caaad8d8c953/include/mbgl/util/constants.hpp#L32) for Web Mercator calculations.

Since different radii are used in different formulas and the errors can sometimes build up, let’s document which is which.